### ISSUE TITLE
Enable UI Testing

### ISSUE THREAD

**Issue by MattLewin** [MattLewin/NumericPicker#6]
**Is your feature request related to a problem? Please describe.**
`NumericPicker` cannot currently be tested using `XCUI*` because the accessibility labels and functionality doesn't work with the Accessibility Inspector. This has the follow-on effect of making it difficult to test the UI of any views that incorporate the control.

I've been trying since Xcode 8 (or maybe Xcode 7), to make this work, but I cannot manage to mimic the functionality that the vanilla `UIPickerView` implements. At various points, I've decided it is impossible, but that hasn't stopped me from spending many hours trying to make it work.

**Describe the solution you'd like**
I'd like `NumericPicker` to behave like any other control with respect to UI automation.

**Describe alternatives you've considered**
It is very easy to set the value of the picker, so it should be easy to simply set its value, and then test the remainder of the UI in which it is embedded.
